assets/js/editor/schema/block-nodes.ts

```diff
@@ -1,89 +1,154 @@
-import {NodeSpec} from "prosemirror-model";
-import {html} from "../../lib/util";
+import { NodeSpec } from 'prosemirror-model';
+import { html } from '../../lib/util';
 
 const createBlockSpec = (type: string, attr = 'id'): NodeSpec => ({
-	attrs: {[attr]: {}},
-	group: "block",
+	attrs: { [attr]: {} },
+	group: 'block',
 	draggable: true,
-	toDOM: node => ["div", {[`data-${type}`]: node.attrs[attr], class: "pm-block"}, `${type}: ${node.attrs[attr]}`],
-	parseDOM: [{tag: `div[data-${type}`, getAttrs: (dom: HTMLElement) => ({[attr]: dom.dataset[type]})}],
-})
+	toDOM: (node) => [
+		'div',
+		{ [`data-${type}`]: node.attrs[attr], class: 'pm-block' },
+		`${type}: ${node.attrs[attr]}`,
+	],
+	parseDOM: [
+		{
+			tag: `div[data-${type}`,
+			getAttrs: (dom: HTMLElement) => ({ [attr]: dom.dataset[type] }),
+		},
+	],
+});
 
-const createGroepBlockSpec = (tagType: string, type: string, attr = 'id'): NodeSpec => ({
-	attrs: {[attr]: {}, naam: {default: ""}},
-	group: "block",
+const createGroepBlockSpec = (
+	tagType: string,
+	type: string,
+	attr = 'id'
+): NodeSpec => ({
+	attrs: { [attr]: {}, naam: { default: '' } },
+	group: 'block',
 	draggable: true,
-	toDOM: node => {
-		const el = html`
-			<div data-${tagType}="${node.attrs[attr]}" class="bb-block"><i class="fa fa-spinner fa-spin"></i></div>`
+	toDOM: (node) => {
+		const el = html` <div
+			data-${tagType}="${node.attrs[attr]}"
+			class="bb-block"
+		>
+			<i class="fas fa-spinner fa-spin"></i>
+		</div>`;
 
 		fetch(`/groepen/${type}/${node.attrs[attr]}/info`)
-			.then(response => response.json())
-			.then(json => {
-				el.dataset.naam = json.naam
-				const ledenDiv = html`
-					<div id="groep-leden-content-2152" class="groep-tab-content GroepPasfotosView" style="height: 212.8px;">`
+			.then((response) => response.json())
+			.then((json) => {
+				el.dataset.naam = json.naam;
+				const ledenDiv = html` <div
+					id="groep-leden-content-2152"
+					class="groep-tab-content GroepPasfotosView"
+					style="height: 212.8px;"
+				></div>`;
 
 				for (const lid of json.leden) {
-					ledenDiv.appendChild(html`<a href="#" title="${lid.naam}" class="lidLink"><img
-						class="pasfoto"
-						src="/profiel/pasfoto/${lid.uid}.jpg"
-						alt="Pasfoto van ${lid.naam}"></a>`)
+					ledenDiv.appendChild(html`<a
+						href="#"
+						title="${lid.naam}"
+						class="lidLink"
+						><img
+							class="pasfoto"
+							src="/profiel/pasfoto/${lid.uid}.jpg"
+							alt="Pasfoto van ${lid.naam}"
+					/></a>`);
 				}
 
-				el.innerHTML = ''
-				el.appendChild(html`
-<div class="bb-groep">
-<div class="groep-samenvatting">
-<div class="float-end"><a class="btn" target="_blank" href="/groepen/${type}/${json.id}/wijzigen"
-title="Wijzig ${json.naam}"><span
-class="fa fa-edit"></span></a></div>
-<h3>${json.naam}</h3>
-${json.samenvatting_html ?? ""}
-</div>
+				el.innerHTML = '';
+				el.appendChild(html` <div class="bb-groep">
+					<div class="groep-samenvatting">
```

Comment:
Weet het niet meer helemaal zeker, maar volgens mij breekt dit de layout van de groep component in prosemirror omdat er nu extra white-space in zit.